assets/ui/components/ArticleBodyHtml.tsx

```diff
@@ -187,49 +215,27 @@ class ArticleBodyHtml extends React.PureComponent<any, any> {
             });
 
         // If Image tags were not updated, then return the supplied html as-is
-        return imageSourcesUpdated ?
-            container.innerHTML :
-            html;
+        return imageSourcesUpdated ? container.innerHTML : html;
     }
 
     render() {
-        const item = this.props.item;
-        const html = this.getBodyHTML(
-            get(item, 'es_highlight.body_html.length', 0) > 0 ?
-                item.es_highlight.body_html[0] :
-                item.body_html
-        );
-
-        if (!html) {
-            return null;
-        }
+        // preview element will be populated in `componentDidMount`
```

Comment:
Moving this functionality outside of `render` to `componentDidMount` means the article body is not updated if a new version is published.  `getBodyHTML` is already memoized so this should not a performance issue

Reply:
it's not using react for rendering to make the embed scripts work